README.md

```diff
@@ -1,2 +1,221 @@
+```
+/mw clan members promote <player>
+/mw clan members demote <player>
+```
+**ONLY ASSIGN RESPONSIBLE AND TRUSTWORTHY PEOPLE AS CLAN LEADERS.** Since they have control over their clan's treasure, it's no fun if they abuse the commands they have available to them and ruin the game for everyone else.
+
+### Changing chat colours
+
+Since Milk Wars needs to display a prefix on a player's username when they're signed in to their clan, it has to take control over the team system. Because of this, there's a special command for allowing players to set their personal chat colours:
+```
+/mw settings setchatcolour <aqua | black | gold | blue ...> <player>
+```
+All vanilla colours are available. This command is intended to be run in command blocks at spawn for players to choose their own colours.
+
+### Creating smart signs
+
+To help display clan statistics you can use smart signs. Smart signs work in a similar way to normal signs, but with the added benefit of giving you more power to format the text and the ability to display automatically updating clan statistics.
+
+#### Variable tags
+
+In order to create a smart sign simply place down a sign and make sure that anywhere in the sign text you include `#MW`. This `#MW` tag tells the plugin that you want this sign to be a smart sign, don't worry about it looking ugly: the tag won't actually be displayed on the sign.
+To insert a variable into the sign you can use variable tags. Like the `#MW` tag these all start with a hash followed by some symbol. The available tags are:
+
+- `#0` - Milk Drinkers kill counter: the amount of times a signed in Milk Drinker has killed a signed in enemy.
+- `#1` - Milk Drinkers capture counter: the amount of times the Milk Drinkers clan have captured the enemy flag.
+- `#2` - Wool Wearers kill counter: the amount of times a signed in Wool Wearer has killed a signed in enemy.
+- `#3` - Wool Wearers capture counter: the amount of times the Wool Wearers clan have captured the enemy flag.
+
+#### Variable tag usage example
+
+<!-- ![Variable tag use example edit message](images/VarTagExampleEdit.png)
+![Variable tag use example after edit](images/VarTagExample.png) -->
+
+<img src="images/VarTagExampleEdit.png" alt="Variable tag example edit message" style="width:250px;"/>
+
+```
+Milk Drinkers:
+#MW
+Kills: #0
+Captures: #1
+```
+Once you finish editing the sign it will display like so:
+
+<img src="images/VarTagExample.png" alt="Variable tag example result" style="width:250px;"/>
+
+Note that in this example the Milk Drinkers clan has 10 kills and has captured the Wool Wearers flag 3 times. Using these smart signs can be very useful as the signs automatically update. So if, for example, the Milk Drinkers got another kill then the sign would update and display `Kills: 11`.
+
+#### Formatting codes
+
+If you've used colour/format codes in vanilla servers then this may be familiar to you. These tags start with a `&` followed by some symbol, the available codes are:
```

Comment:
```suggestion
If you've used colour/format codes in servers before then this may be familiar to you. These tags start with a `&` followed by some symbol, the available codes are:
```